chcemvediet/apps/inforequests/admin.py

```diff
@@ -13,6 +13,51 @@
 from .models import Inforequest, InforequestDraft, InforequestEmail, Branch, Action
 
 
+class NestedInforequestemailAdminMixin(object):
```

Comment:
`DeleteNestedInforequestEmailAdminMixin` - velke "E" v Email, kedze je to nazov classy, tam meme velke "E", nie nazov instancie. A "Delete" prefix, nech je jasne, ze to sluzi na ich deletovanie.

Reply:
:ok: